src/lizaalert/courses/models.py

```diff
@@ -441,14 +441,14 @@ class Subscription(TimeStampedModel):
     enabled - признак активности записи на курс.
     """
 
-    class Flag(models.TextChoices):
-        ACTIVE = 1, "Запись активна"
-        INACTIVE = 0, "Запись не активна"
+    class Status(models.TextChoices):
+        ENROLLED = "Enrolled", "Запись активна"
```

Comment:
договорились о snake_case

```suggestion
        ENROLLED = "enrolled", "Запись активна"
```